osquery/config/packs.cpp

```diff
@@ -216,7 +216,8 @@ const std::string& Pack::getVersion() const {
 }
 
 bool Pack::shouldPackExecute() {
-  return (valid_ && checkDiscovery());
+  executing_ = (valid_ && checkDiscovery());
+  return executing_.load();
```

Comment:
i could see doing a `return isExecuting();` here? it doesn't really matter though